recipes/ogh/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.1.5a0" %}
+
+package:
+  name: ogh
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/o/ogh/ogh-{{ version }}.tar.gz
+  sha256: ccea38e805f6279e7f69a53caa4ae57baa0b742d7cc4ad6d80eec2110a4e7ed3
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - pandas
+    - dask
+    - descartes
+    - geopandas
+    - beautifulsoup4
+    - basemap
+    - wget
```

Comment:
We need to package the python module `wget`. The `wget` present in the default channel is not the python module but the actual `wget`. I'll try to submit a recipe soon.